databroker/_drivers/mongo_normalized.py

```diff
@@ -350,6 +350,9 @@ def _get_serializer(self):
         from suitcase.mongo_normalized import Serializer
         return Serializer(self._metadatastore_db, self._asset_registry_db)
 
+    def stats(self):
+        "Access mongo ``db.collection.stats()`` method"
```

Comment:
It might not be clear to the uninitiated that this refers to MongoDB client code, not Python code. Would it be worth clarifying that, and perhaps making the top line of the docstring a more direct description, something like “Access MongoDB storage statistics for this database”?

Reply:
Thanks for the review!  Changes on the way...